### ISSUE TITLE
Sensory Motion Capture in Godot

### ISSUE THREAD

**Issue by meisamsafavi** [hoontee/godot-ni-mate-motion-capture#10]

**Comment by hoontee** [comment]
Considering I don't have access to the hardware, I probably won't be much use to your project. However, if you can provide a motion capture sample file and a way for me to transmit it, I will be able to create a new plugin for your purposes.

**Comment by meisamsafavi** [comment]
Thanks for your reply. 
For preparing the data for your test, I need to know that which type of data and which format should I generate. 
The data type of IMU sensors is Quaternion and it's representing the same Pitch,Roll,Yaw for rotation that you're dealing with. 
Does Ni Mate software generate the same type of data or a different one?
To better understand what I mean, please check out the following video:

https://user-images.githubusercontent.com/74149722/119440805-3f23dd80-bd3a-11eb-9244-23fee79d58d0.mp4

In other words the goal is to play and act with our exact body movement (real time) which is capturing with IMU sensors.
Hope that I gave you a clear image.
Kind Regards
Mason

**Comment by hoontee** [comment]
Sorry for the wait.

As long as the data is transferred over the network, then there will be minimal changes necessary for the addon. The current version only supports joint origins, but it can be modified to support quaternions for IMU sensors only.

**Comment by meisamsafavi** [comment]
Hey again,
sorry for delay,
thanks for your reply,
would this feature (IMU data) be added to this addon or I should myself add it?

**Comment by hoontee** [comment]
So long as you provide me with a way to transmit a sample of IMU data over the network then I will be able to modify the addon myself.

**Comment by meisamsafavi** [comment]
Hi
Transmitting the IMU data over the network may be a little bit challenging for first steps.
I will attach a sample of 2 IMU Quaternion data in text format so you can check it out.
[Quaternion Data.txt](https://github.com/hoontee/godot-ni-mate-motion-capture/files/6721006/Quaternion.Data.txt)
